content/en/real_user_monitoring/browser/tracking_user_actions.md

```diff
@@ -75,6 +75,22 @@ The RUM library uses various strategies to get a name for click actions. If you
   Enter a valid email address
 </div>
 ```
+
+Starting with [version 2.16.0][3], with the `actionNameAttribute` initialization parameter, you can specify your own attribute that will be used to name the action. For example:
```

Comment:
Should we mention that `data-dd-action-name` takes precedence over the custom attribute if both exists on the element?

Reply:
sure